### ISSUE TITLE
Cannot find module 'ncp' error - jscpd & node

### ISSUE THREAD

**Issue by wesleyscholl** [kucherenko/jscpd#585]
**Describe the bug**
When running ```jscpd /src``` or ```npx jscpd /src``` node produces the follow error:

```
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'ncp'
Require stack:
- /Users/wscholl/.config/yarn/global/node_modules/@jscpd/html-reporter/dist/index.js
- /Users/wscholl/.config/yarn/global/node_modules/jscpd/dist/init/reporters.js
- /Users/wscholl/.config/yarn/global/node_modules/jscpd/dist/index.js
- /Users/wscholl/.config/yarn/global/node_modules/jscpd/bin/jscpd
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/wscholl/.config/yarn/global/node_modules/@jscpd/html-reporter/dist/index.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/wscholl/.config/yarn/global/node_modules/@jscpd/html-reporter/dist/index.js',
    '/Users/wscholl/.config/yarn/global/node_modules/jscpd/dist/init/reporters.js',
    '/Users/wscholl/.config/yarn/global/node_modules/jscpd/dist/index.js',
    '/Users/wscholl/.config/yarn/global/node_modules/jscpd/bin/jscpd'
  ]
}

Node.js v18.15.0
```

**To Reproduce**
1. Install jscpd ```yarn add jscpd global```
2. Open project directory
3. Run ```jscpd /src``` or ```npx jscpd /src```

**Expected behavior**
jscpd should run on the working directory.

**Screenshots**

<img width="830" alt="Screenshot 2023-05-01 at 9 18 23 AM" src="https://user-images.githubusercontent.com/128409641/235457160-4ac5fe23-10ae-4477-af51-29f3ecc11bf5.png">

**Laptop: MacBook Pro 2.3 GHz 8-Core Intel Core i9**
 - OS: MacOS
 - OS Version: Ventura 13.3.1
 - NodeJS Version: v18.15.0
 - jscpd Version: 3.5.6

**Additional context**
Attempted to install ncp globally and within the project directory, restarted terminal, produces the same error. I'm thinking it might be a node version issue. Any ideas? Looking forward to using this software, thanks.

**Comment by kucherenko** [comment]
fixed at jscpd@3.5.8

**Comment by wesleyscholl** [comment]
Thanks!
